app/validators/program_validator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ProgramValidator < ActiveModel::Validator
+  def self.non_unique_error_msg(record)
```

Comment:
Readability suggestion:  Can we turn this into a private instance method and move it below `validate`.
```
def validate(record)
  # code
end

private

def non_unique_error_msg(record)
  # code
end
```

Reply:
This method is used in the unit test

Reply:
changed